util/stmtsummary/statement_summary_test.go

```diff
@@ -42,134 +45,442 @@ func TestT(t *testing.T) {
 	TestingT(t)
 }
 
-// Test stmtSummaryByDigest.AddStatement
+// Test stmtSummaryByDigest.AddStatement.
 func (s *testStmtSummarySuite) TestAddStatement(c *C) {
 	s.ssMap.Clear()
 
-	// First statement
+	mu := struct {
+		sync.Mutex
+		BackoffTypes []fmt.Stringer
+	}{
+		BackoffTypes: make([]fmt.Stringer, 0),
+	}
+	tables := []stmtctx.TableEntry{{"db1", "tb1"}, {"db2", "tb2"}}
+	indexes := []string{"a"}
+
+	// first statement
 	stmtExecInfo1 := &StmtExecInfo{
```

Comment:
Could we use `generateAnyExecInfo()` directly?

Reply:
Addressed.